crowdnotifier-sdk/src/main/proto/backend_v3.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+package org.crowdnotifier.android.sdk.model;
+
+message QRCodeTrace {
+  uint32 version = 1;
+  bytes qrCodePayload = 2;
```

Comment:
Is that in bytes and not of type `QRCodePayload` for clearer marshalling?

Reply:
jep

Reply:
Good question. Sounds like a good idea to me to change this to QRCodePayload. @martinalig @wouterl  @ubfelix  What do you guys think?

Reply:
Ok. It seems like I haven't refreshed my page. I will change this then.

Reply:
So for clarity: I'd prefer to keep here `bytes` so that we can rule out any later tricky issues with inconsistent protobuf implementations. Then again, that wasn't a problem before, so maybe I'm worried for nothing.

Reply:
Ah okay. Let's keep it bytes then.